services/navigation/builtin/builtin.go

```diff
@@ -88,8 +120,10 @@ type builtIn struct {
 
 	base           base.Base
 	movementSensor movementsensor.MovementSensor
+	motion         motion.Service
+	obstacles      []*spatialmath.GeoObstacle
 
-	mmPerSecDefault         float64
+	metersPerSecDefault     float64
```

Comment:
could we also change these from having default in their name too?  We do have defaults at the top of the file so this is a little confusing